celest/encounter/_window_handling.py

```diff
@@ -315,12 +320,13 @@ def save(self, fname: str, delimiter: Literal[",", "\\t"]) -> None:
         ]
 
         for i, window in enumerate(np_data):
-            
+
             out_data[i + 1, 0] = window.coor[0]
             out_data[i + 1, 1] = window.coor[1]
             out_data[i + 1, 2] = window.start
             out_data[i + 1, 3] = window.end
             out_data[i + 1, 4] = window.duration
             out_data[i + 1, 5] = window.type
 
```

Comment:
This should be an enum, or you should define an interface to use out_data, and employ that here. This seems like an opportunity for indexing bugs